concordia/views.py

```diff
@@ -1128,17 +1129,40 @@ def reserve_asset(request, *, asset_pk):
 
     timestamp = now()
 
-    # First clear old reservations, with a grace period:
+    # First clear old reservations, with a grace period.
     cutoff = timestamp - (
         timedelta(seconds=2 * settings.TRANSCRIPTION_RESERVATION_SECONDS)
     )
 
     with connection.cursor() as cursor:
-        cursor.execute(
-            "DELETE FROM concordia_assettranscriptionreservation WHERE updated_on < %s",
+        rows_to_release = cursor.execute(
```

Comment:
I think we can do this in one query using [`DELETE … RETURNING`](https://www.postgresql.org/docs/9.6/sql-delete.html)

Reply:
Oh that would be cool. I'll try that tomorrow